src/styles/01-atoms/_icons/_icons.scss

```diff
@@ -3,19 +3,26 @@
     width: 24px;
     fill: currentColor;
 
-    &--up {
+    &--up,
+    &--rotate-180 {
+        @include space-inline-xxs;
         transform: rotate(180deg);
     }
-
-    &--right {
+    &--right,
```

Comment:
I would love to know if this is being used in places that aren't buttons.  If not, I'd be for removing it.  This would be a breaking change, but since we're making some big changes already in DS, I think it's better to do it now than later.

Reply:
@hokei I updated the scss to avoid introducing breaking changes to the styling

Reply:
To answer your question @hokei, links.